stellargraph/__init__.py

```diff
@@ -21,11 +21,23 @@
 
 __all__ = [
     "data",
+    "calibration",
+    "ensemble",
+    "interpretability",
+    "losses",
     "layer",
     "mapper",
     "utils",
+    "custom_keras_layers",
+    "expected_calibration_error",
```

Comment:
Yeah, only `StellarGraph` and `StellarDiGraph` really makes sense to me to export at the top-level, it's not clear to me why the rest of the classes/functions need special treatment. One reason for their previous inclusion might've been because they were under `utils` and we didn't want users to have to write `from stellargraph.utils.calibration import IsotonicCalibration` etc.. but now with `utils` gone maybe it makes sense to just remove them from the top-level export